.github/workflows/lint.yml

```diff
@@ -0,0 +1,39 @@
+name: Lint JavaScript
+
+on:
+    pull_request:
+        types: [opened, synchronize]
+
+jobs:
+    lint:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v2
+
+            - uses: actions/cache@v2
+              with:
+                  path: ~/.npm
+                  key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+                  restore-keys: |
+                      ${{ runner.os }}-node-
+
+            - name: Setup Node
+              uses: actions/setup-node@v1
+              with:
+                  node-version: "16.x"
+
+            - run: npm ci
```

Comment:
Does this work without a package.json in the repo?

Reply:
Probably not, but I committed the `package.json` directly to `main` (oops) so it's already there.